migrations/20250818_add_start_of_week.go

```diff
@@ -0,0 +1,41 @@
+package migrations
```

Comment:
You're actually not gonna need this migration, thanks to GORM's [auto-migrate](https://gorm.io/docs/migration.html#Auto-Migration) feature. Will remove it when merging.